src/Catalog.Api/Controllers/GraphQL/CatalogController.cs

```diff
@@ -0,0 +1,94 @@
+﻿using GraphQL.AspNet.Attributes;
+using GraphQL.AspNet.Controllers;
+using Catalog.Domain.Entities;
+using Catalog.Application.Services;
+using Catalog.Api.Requests;
+using Catalog.Api.Mappers;
+
+namespace Catalog.Api.Controllers.GraphQL
+{
+    public class CatalogController : GraphController
```

Comment:
`All actions should be accessible only with permissions/roles used in Catalog service from previous modules.`
Is this requirement fulfilled? We can discuss this during a call